thrift/sampling.thrift

```diff
@@ -35,17 +35,25 @@ struct RateLimitingSamplingStrategy {
     1: required i16 maxTracesPerSecond
 }
 
-// OperationSamplingStrategy is the strategy used per operation.
+// OperationSamplingStrategies samples traces per operation with a guaranteed lower bound per second. Once
```

Comment:
* nit: I think "PerOperation" is a more accurate name (here and in field 4 below, but not in L47)
* nit: comment should read "PerOperationSamplingStrategies defines a sampling strategy per each operation name in the service..."